pennylane/decomposition/symbolic_decomposition.py

```diff
@@ -131,6 +131,49 @@ def adjoint_controlled_decomp(*_, base, **__):
     )
 
 
+def _adjoint_pow_resource(base_class, base_params):  # pylint: disable=unused-argument
+    """Resources of the adjoint of the power of a gate."""
+    base, base_params, z = base_params["base_class"], base_params["base_params"], base_params["z"]
+    return {pow_resource_rep(base, base_params, z): 1}
```

Comment:
Don't we loose the adjoint here? :thinking:

Reply:
This is exclusively for the same-type adjoint ops, I'll modify the docstring to highlight this.

Reply:
Thanks! I got confused by the different cases that can happen and thought that the decomposition was calling `.adjoint()` because it creates a different op, but it's just to get back the correct same-type op (e.g. with parameter sign flipped or so).

Got it :)